File: 프로그래머스/2/42578. 의상/의상.swift
import Foundation

func solution(_ clothes:[[String]]) -> Int {
    
    var d: [String: Int] = [:]
    
    for cloth in clothes {
        d[cloth[1], default: 0] += 1
    }
    
    var answer = 1
    
    for v in d.values {
        answer *= v + 1
    }
    
    return answer - 1
}

File: 프로그래머스/2/42578. 의상/의상.py
def solution(clothes):
    
    d = {}
    
    for cloth in clothes:
        d[cloth[1]] = d.get(cloth[1], 0) + 1
        
    answer = 1
    for k, v in d.items():
        answer *= v + 1
        
    return answer - 1